pkg/virt-launcher/virtwrap/converter/BUILD.bazel

```
@@ -47,6 +47,12 @@ go_test(
         "converter_test.go",
     ],
     embed = [":go_default_library"],
+    embedsrcs = [
+        "embed/template_domain_ppc64le.xml.tmpl",
+        "embed/template_domain_x86_64.xml.tmpl",
+        "embed/template_domain_arm64.xml.tmpl",
+        "embed/template_domain_x86_64_root.xml.tmpl",
```

nit: I would omit `template_` prefix. IMHO the intension is clear from the `.tmpl` extensions,
```suggestion
        "embed/domain_ppc64le.xml.tmpl",
        "embed/domain_x86_64.xml.tmpl",
        "embed/domain_arm64.xml.tmpl",
        "embed/domain_x86_64_root.xml.tmpl",
```

---

Agreed. I'll change it.